packages/curve-ui-kit/src/features/connect-wallet/lib/types.ts

```diff
@@ -38,3 +38,12 @@ export type Libs = {
 export type LibKey = keyof Libs
 
 export { CurveChainId, LlamaChainId, CurveNetworkId, LlamaNetworkId }
+
+export type Network = {
```

Comment:
How is this related to the other network definitions elsewhere? :skull:

Reply:
Cheers, got lost in the many types; using that now